test/vmtester/vmtester.hpp

```diff
@@ -1,23 +1,18 @@
 // EVMC: Ethereum Client-VM Connector API
 // Copyright 2018-2019 The EVMC Authors.
 // Licensed under the Apache License, Version 2.0.
-
 #pragma once
 
 #include <evmc/evmc.h>
-
 #include <gtest/gtest.h>
 
-evmc_instance* get_vm_instance();
-
 class evmc_vm_test : public ::testing::Test
 {
-protected:
-    evmc_instance* vm = nullptr;
+public:
+    /// The raw pointer to the loaded VM instance.
+    /// The C API is used to allow more sophisticated unit tests.
```

Comment:
As mentioned here, we want to access to C API for better coverage.

Reply:
Still maybe better to declare `evmc::vm` as a static member of the class close to `vm`? Then it's clearer that they're coupled.

Reply:
Or maybe I would just not use `vm` at all then, and destroy it manually in `evmc_vm_test` destructor

Reply:
I moved them close together. Using the destructor is not an option because destructor is called for every test case.